common/src/main/java/net/william278/huskhomes/util/UnsafeBlocks.java

```diff
@@ -49,6 +52,19 @@ public boolean isUnsafe(@NotNull String blockId) {
         return unsafeBlocks.contains(blockId);
     }
 
+    /**
+     * Returns if the block, by provided identifier, is unsafe to stand in
+     *
+     * @param blockId The block identifier (e.g. {@code minecraft:stone})
+     * @return {@code true} if the block is on the unsafe blocks list, {@code false} otherwise
+     */
+    public boolean isUnsafeToOccupy(@NotNull String blockId) {
+        if (!blockId.startsWith(Key.MINECRAFT_NAMESPACE + ":")) {
+            blockId = Key.MINECRAFT_NAMESPACE + ":" + blockId;
+        }
```

Comment:
Could we deduplicate this stuff into a private `formatBlockId` method here?

Reply:
https://github.com/WiIIiam278/HuskHomes/pull/445/commits/1941b3785ed384bac38c1b426ac265641ff0fb5e